packages/dashboard/src/components/schedule-visualizer/index.tsx

```diff
@@ -61,6 +63,7 @@ export interface ScheduleVisualizerProps extends React.PropsWithChildren<{}> {
    * default: 'normal'
    */
   mode?: 'normal' | 'negotiation';
+  zoom?: number | undefined;
```

Comment:
Instead of adding this prop, you may consider forwarding the ref of `LMap`. This component already has a ref callback, so you need to manually set the ref

```tsx
export const ScheduleVisualizer = React.forwardRef((props, ref) => {
  return (
    <LMap
      ref={(map) => {
        ...
        ref.current = map;
     }
     </LMap>
    )
});
```

---

Actually, I realized this props is not used?